README.md

```diff
@@ -1,24 +1,35 @@
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-Creates and configures an S3 bucket for storing logs from various AWS
-services and enables CloudTrail on all regions. Logs will expire after a
-default of 90 days. Includes support for sending CloudTrail events to a
-CloudWatch Logs group.
+Supports two main uses cases:
 
-Logging from the following services is supported:
+1. Creates and configures a single private S3 bucket for storing logs from various AWS services, which are nested as bucket prefixes, and enables CloudTrail on all regions. Logs will expire after a default of 90 days, with option to configure retention value. Includes support for sending CloudTrail events to a CloudWatch Logs group.
```

Comment:
This description should be updated. The module no longer enables CloudTrail or configures CloudTrail to write to CloudWatch Log groups since that's really not what this module was designed to do (it should be a separate module).